hack/setup.sh

```diff
@@ -11,3 +11,6 @@ install_go_tools
 
 ${IDPBUILDER} create \
     -p ${HACK_DIR}/config/cnoe-packages
+
+chmod +x ${HACK_DIR}/add-vclusters.sh
```

Comment:
If you run this command and then commit the file with +x set this should not be needed.